docs/reference/plugins/minimator.rst

```diff
@@ -0,0 +1,22 @@
+.. reference-gen: minimator
+
+Minimator Simulator
+===================
+
+.. todo:: Describe the minimator simulator.
+
+Defaults
+--------
+
+The following help can be viewed with :command:`cloe-engine usage basic`:
+
+.. literalinclude:: basic.yaml
+   :language: yaml
+
+JSON Schema
+-----------
+
+The following can be viewed with :command:`cloe-engine usage --json basic`:
+
+.. literalinclude:: basic_schema.json
```

Comment:
should be `minimator` instead of `basic`

Reply:
Oops.